core/src/main/java/io/questdb/griffin/FunctionParser.java

```diff
@@ -76,17 +73,11 @@ public FunctionParser(CairoConfiguration configuration, FunctionFactoryCache fun
     }
 
     @NotNull
-    public static ScalarFunction createColumn(
-            int position,
-            CharSequence name,
-            RecordMetadata metadata
-    ) throws SqlException {
-        final int index = metadata.getColumnIndexQuiet(name);
-
+    public static ScalarFunction createColumn(int position, CharSequence name, RecordMetadata metadata) throws SqlException {
+        int index = metadata.getColumnIndexQuiet(name);
```

Comment:
final ?

Reply:
final :D that is a cosmetic. All it does is to make you think that the code is immutable. I view it mostly as a waste of space unless the method is 45 cm in length (of my screen) and I have to scroll around to figure out if the variable is immutable.... I think this debate is equivalent to:

# Option A)

```
if (null == variable) {
}
```

# Option B)

```
if (variable == null) {
}
```

matter of preference.